Sources/SmithyCodegenCore/AST/ASTMember.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
This and the below `AST...` files are just containers for the JSON data read from the AST file.